testsuite/MDAnalysisTests/core/test_groups.py

```diff
@@ -1466,3 +1466,85 @@ def test_decorator(self, compound, pbc, unwrap):
                 self.dummy_funtion(compound=compound, pbc=pbc, unwrap=unwrap)
         else:
             assert_equal(self.dummy_funtion(compound=compound, pbc=pbc, unwrap=unwrap), 0)
+
+
+@pytest.fixture()
+def tpr():
+    return mda.Universe(TPR)
+
+
+class TestGetConnections(object):
```

Comment:
Could we get some tests to cover at the very least the two cases that were outlined in #1264 and #2821?

Reply:
Do you mean those specific files and selections, or do you mean using the .bonds and .dihedrals attributes? The latter will have to wait until 2.0